lib/Runtime/Types/PathTypeHandler.h

```diff
@@ -6,16 +6,62 @@
 
 namespace Js
 {
+    typedef uint8 SetterSlotIndex;
```

Comment:
Maybe explicit naming this PathTypeHandlerSettingSlotIndex since this is only for PathTypeHandlers?